picard/webservice.py

```diff
@@ -314,22 +310,22 @@ def _process_reply(self, reply):
             reply.deleteLater()
 
     def get(self, host, port, path, handler, xml=True, priority=False,
-            important=False, mblogin=False, cacheloadcontrol=None, refresh=False):
+            important=False, mblogin=False, cacheloadcontrol=None, refresh=False, queryargs=None):
```

Comment:
It could an object having QUrl as parent, since we need few more properties on it.
I didnt look into details, but it seems to me those methods have too much arguments now, so anything simplifying this would be good imho.